CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Cast
 
+#### Added
+- Commands that commit transactions now display links to block explorers. When in human-readable mode, `invoke`, `declare`, `deploy`, `account create` and `account deploy` will display additional information with an url.
```

Comment:
Super nit, but probably doesn't need to mention that it's in human readable mode.

Reply:
I think it's worth emphasising. I can write that this behaviour is disabled in JSON mode instead